content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Collect Google Cloud Logs with a Pub/Sub Push Subscription
+kind: documentation
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "/logs/explorer/#visualize"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "/logs/log_configuration/processors"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+---
+
+<div class="alert alert-danger">
+This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups. Use a <strong>Pull</strong> subscription with the Datadog Dataflow template to forward your Google Cloud logs to Datadog instead. See <a href="https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection" target="_blank">Log collection</a> on the Google Cloud integration page for instructions.
```

Comment:
Nit:

I was a bit surprised that the logs guide index doesn't include a link to the new log forwarding mechanism (Dataflow). OTOH, it'd be weird to have an integrations page in the table. Or maybe we can add to the table with the correct anchor `#log-collection` ? 

Your call, just wanted to mention.

Reply:
Thanks for bringing that up as well, it makes sense and I've added the link in [0c8d91b](https://github.com/DataDog/documentation/pull/19467/commits/0c8d91b4dd4b93d757a0b221774f42bca3ce708a).